### ISSUE TITLE
.Net 4.7.1 supports .Net Standard 2.0

### ISSUE THREAD

**Issue by jvecchioli** [kblok/puppeteer-sharp#750]
Hi guys,

We have been testing PuppeteerSharp on a machine that only had .Net 4.7.1 installed, and it perfectly works as the Framework supports .Net Standard 2.0.

Here is the Release Notes from Microsoft: https://blogs.msdn.microsoft.com/dotnet/2017/10/17/announcing-the-net-framework-4-7-1/

Just wanted to let you know as it could be mentionnned in the Readme.

Thanks,
J

**Comment by kblok** [comment]
Hey @jvecchioli. The readme states:
>The minimum platform versions are .NET Framework 4.6.1 and .NET Core 2.0. 

Am I missing something?

**Comment by jvecchioli** [comment]
Hi @kblok,

I have been testing it on a Win Server 2016 that has by default .Net 4.6.2, and it does not work (it throws an Exception with the following error message: _One or more errors occurred._). This is most likely due to a Dll not found issue (I have not verified it with the Fusion Logs).

Therefore, .Net Standard 2.0 does not seem to be fully supported (at least some of the dependencies of PuppeteerShar might not be). 

The only thing that could fix it was the upgrade to .Net 4.7.1+ (in fact, I tested with .Net 4.7.2, but I am pretty sure it also works with 4.7.1).

In short, I am not sure that the minimum platform version of .Net 4.6.1 is a correct statement - at least, on my end, I was unable to get it to work with only 4.6.2 on Windows Server 2016.

**Comment by kblok** [comment]
@jvecchioli is there any chance you can see, what's missing on Windows Server 2016?

**Comment by kblok** [comment]
Closed due to inactivity. Feel free to reopen it if needed.
